### ISSUE TITLE
Crash after enable backup contacts 

### ISSUE THREAD

**Issue by aleister09** [nextcloud/android#885]
### Steps to reproduce
1. Open App
2. Enable Contacts Backup
3. Logout
4. Close App
5. Open App

### Environment data
Android version: 7.1.2

Device model: Nexus 5X

Stock or customized system:

Nextcloud app version:Master

Nextcloud server version:12

### Logs

```
04-24 15:15:57.020 25218-25218/com.nextcloud.client E/AndroidRuntime: FATAL EXCEPTION: main
                                                                      Process: com.nextcloud.client, PID: 25218
                                                                      java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.lang.NullPointerException: Attempt to read from field 'java.lang.String android.accounts.Account.name' on a null object reference
                                                                          at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2666)
                                                                          at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2727)
                                                                          at android.app.ActivityThread.-wrap12(ActivityThread.java)
                                                                          at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1478)
                                                                          at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                          at android.os.Looper.loop(Looper.java:154)
                                                                          at android.app.ActivityThread.main(ActivityThread.java:6121)
                                                                          at java.lang.reflect.Method.invoke(Native Method)
                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
                                                                       Caused by: java.lang.NullPointerException: Attempt to read from field 'java.lang.String android.accounts.Account.name' on a null object reference
                                                                          at com.owncloud.android.ui.activity.ContactsPreferenceActivity.startContactBackupJob(ContactsPreferenceActivity.java:306)
                                                                          at com.owncloud.android.ui.activity.FileActivity.checkContactsBackupJob(FileActivity.java:252)
                                                                          at com.owncloud.android.ui.activity.FileActivity.onCreate(FileActivity.java:167)
                                                                          at com.owncloud.android.ui.activity.FileDisplayActivity.onCreate(FileDisplayActivity.java:168)
                                                                          at android.app.Activity.performCreate(Activity.java:6682)
                                                                          at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
                                                                          at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2619)
                                                                          at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2727) 
                                                                          at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                                                                          at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1478) 
                                                                          at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                          at android.os.Looper.loop(Looper.java:154) 
                                                                          at android.app.ActivityThread.main(ActivityThread.java:6121) 
                                                                          at java.lang.reflect.Method.invoke(Native Method) 
                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889) 
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779) 
```

**Comment by mario** [comment]
I assume "logout" means removing the account from the application?

**Comment by aleister09** [comment]
Yes, you are right!

**Comment by tobiasKaminsky** [comment]
Fixed in #874
